packages/dashboards/tests/acceptance/dashboards-test.js

```diff
@@ -573,4 +573,59 @@ module('Acceptance | Dashboards', function(hooks) {
 
     assert.dom('.navi-dashboard__edit-title').hasText('EFGH', 'Keeps updated text after save');
   });
+
+  test('modifying dashboard filters, navigating away, and coming back to modifications', async function(assert) {
+    assert.expect(3);
+
+    let dataRequests = [];
+    server.urlPrefix = `${config.navi.dataSources[0].uri}/v1`;
+    server.pretender.handledRequest = (verb, url, req) => {
```

Comment:
Could this be:
```js
server.get('/data/*path', () => {
```

Reply:
Probably, I remembered seeing it used here so I opted to follow that
 https://github.com/yahoo/navi/blob/94509edd5bfba8daac0f62d7cc49b22a5b8be227/packages/dashboards/tests/acceptance/dashboard-filter-test.js#L110-L118